README.md

```diff
@@ -174,6 +174,25 @@ project("web") {
 }
 ```
 
+## FAQ
+
+Q: Why the `flow-server-production-mode.jar` file is missing in the WAR artifact built for production?
+
+A: That jar file is actually not really needed. Its sole purpose is to set the `productionMode` `web.xml` context-param, however
+   we're telling this information to Vaadin in a different way. Please see next question for more details.
+
+Q: How is Vaadin configured for production mode?
+
+A: Vaadin is configured via the `META-INF/VAADIN/config/flow-build-info.json` file.
+   When the Plugin is set for production mode, either via `-Pvaadin.productionMode` or via `vaadin { productionMode = true }`,
+   it will set the `productionMode` JSON property to true in the `flow-build-info.json` file.
+   That will then tell Vaadin to run in production mode.
+
+Q: How can I verify that the WAR file has been built correctly for production mode?
+
+A: Please read the [Development vs production mode](https://mvysny.github.io/Vaadin-the-missing-guide/)
```

Comment:
If the question is common and relevant, could we either answer it here or in vaadin.com (by improving docs or the FAQ), instead of bashing our own documentation?

Reply:
Good question. I learned it's much faster to write own docs. And the topic pops up from time to time on EC, that's why I decided to write this info. I'd propose to leave this as-is; if this ever gets included into official docs I'll happily change the link.

Reply:
Opened a PR for the main docs: https://github.com/vaadin/flow-and-components-documentation/pull/1341

Reply:
No activity on the PR, moving forward.